lib/pretty_text.rb

```diff
@@ -279,10 +292,6 @@ def self.escape_emoji(title)
 
   def self.cook(text, opts = {})
     options = opts.dup
-
-    # we have a minor inconsistency
-    options[:topicId] = opts[:topic_id]
```

Comment:
This was written so long ago that I'm not even sure we have that inconsistency anymore.

Reply:
I support removing it but we should remember we did this if any regressions pop up.